charts/cluster-api-cluster-openstack/templates/workload.yaml

```diff
@@ -107,6 +107,14 @@ spec:
         path: {{ $file.path }}
         permissions: "0600"
       {{- end }}
+      - path: /etc/kubernetes/patches/kubeletconfiguration0+strategic.json
+        owner: "root:root"
+        permissions: "0644"
+        {{- with $config := $pool.kubelet }}
```

Comment:
This doesn't actually work when you test it...

Needs to be
```
{{- with ... -}}
{{- $_ := ... -}}
{{- $_ := ... }}
```

please note the trailing whitespace chomping, otherwise you get
```
      - path: /etc/kubernetes/patches/kubeletconfiguration0+strategic.json
        owner: "root:root"
        permissions: "0644"content: {"apiVersion":"kubelet.config.k8s.io/v1beta1","foo":"bar","kind":"KubeletConfiguration"}
```

Reply:
Formatting fixed, and I've tested this with multiple pool definitions along with differing options.